src/runner/test-run/index.js

```diff
@@ -11,6 +14,9 @@ const TEST_RUN_TEMPLATE        = read('../../client/test-run/index.js.mustache')
 const IFRAME_TEST_RUN_TEMPLATE = read('../../client/test-run/iframe.js.mustache');
 
 
+var ensureDir = promisify(mkdirp);
+
+
 export default class TestRun extends Session {
     constructor (test, browserConnection, opts) {
         super(path.dirname(test.fixture.path));
```

Comment:
Can we extract `path.dirname(test.fixture.path)` to the `uploadsRoot` var for the clarity?